tasmota/settings.h

```diff
@@ -573,4 +573,9 @@ typedef union {
   ADC_MODE(ADC_VCC);                       // Set ADC input for Power Supply Voltage usage
 #endif
 
+// Settings re-purposed for device groups
+#define device_groups_enabled  ws_clock_reverse   // SetOption16 - Enable device groups
+#define device_group_share_in  domoticz_sensor_idx[0]    // Bitmask of device group items imported
+#define device_group_share_out domoticz_sensor_idx[1]    // Bitmask of device group items exported
```

Comment:
Theo, is it ok to re-purpose those parameters, or should we assign a fresh new location?

Reply:
Not OK. It removes Domoticz functionality.

As you suggested, let's reserver a new setoption and two uint16_t in Settings area.

Reply:
Do we need Device Groups? Does adding this flood UDP traffic?

Reply:
That's why I'm curious about broadcast/unicast packets